test/test_base58.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) 2018 John Newbery
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+"""Test base58.py"""
+import unittest
+
+import base58
+from binascii import unhexlify
+
+TEST_CASES = [
+    ["", ""],
+    ["61", "2g"],
+    ["626262", "a3gV"],
+    ["73696d706c792061206c6f6e6720737472696e67", "2cFupjhnEsSn59qHXstmK2ffpLv2"],
+    ["00CF00816E4BE7DBC1C3DF0C2B0BE2B77E4AD99A14111E6C6F", "1KsXUqbYDGz5gvBL2mo6as5KqKueRWc2Wr"],
+    ["00000000000000000000", "1111111111"],
+    # ["  1234", ""]
```

Comment:
Remove this line?

Reply:
I can. I didn't know how the encoding should deal with unexpected edge cases.

Reply:
bitcoind removes leading and trailing spaces when decoding base58 strings. This library only has bytes->base58 string, so leading spaces doesn't really make sense.

Reply:
Gotcha!